ui/desktop/src/sessions.ts

```diff
@@ -40,7 +41,7 @@ export interface SessionMessage {
 export interface SessionDetails {
   session_id: string;
   metadata: SessionMetadata;
-  messages: SessionMessage[];
```

Comment:
i think we can remove this - https://github.com/block/goose/blob/3b36591cb55e291d9de46ec98df025c317f0d13e/ui/desktop/src/sessions.ts#L31